### ISSUE TITLE
winning the pot at showdown

### ISSUE THREAD

**Issue by rafanoronha** [rafanoronha/scala-poker#23]
"Once a betting round is finished another card is dealt on the board, and another round of betting follows until the showdown takes place after the river.
(...) The player with the best shown hand will win the pot".
http://www.ultimatepoker.com/how-to-play-online-poker/poker-tips/

Need to enable a hand to **run through** pre-flop to showdown.
